<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scene</class>
 <widget class="QWidget" name="Scene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="styleSheet">
          <string notr="true">QSlider{
	height: 32px;
	margin-left: 16px;
	margin-right: 16px;
}

QSlider::groove:horizontal {
	border: 1px solid #bbb;
	background: white;
	height: 9px;
	border-radius: 5px;
}

QSlider::sub-page:horizontal {
	background: rgb(85, 170, 255);
	border: 1px solid #777;
	height: 9px;
	border-radius: 5px;
}

QSlider::add-page:horizontal {
	background: #fff;
	border: 1px solid #777;
	height: 9px;
	border-radius: 5px;
}

QSlider::handle:horizontal {
	background: transparent;
	image:url(:/icons/diamond.png);
	width: 32px;
	height: 32px;
	margin-top: -20px;
	margin-bottom: -20px;
	margin-right: -16px;
	margin-left: -16px;
}</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
